ENTERPRISE-MIB

```diff
@@ -0,0 +1,151 @@
+CLEARWATER-ALARMS-MIB DEFINITIONS ::= BEGIN
```

Comment:
ENTERPRISE-MIB is too generic (imagine the chaos if 20 vendors all gave you an ENTERPRISE-MIB file!).